speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/conformationmenu.h

```diff
@@ -0,0 +1,28 @@
+/***************************************************************************
+
+    file                 : conformation menu.h
+    created              : tuesday Mar 10 10:48:22 CET 2022
+    copyright            : (C) 2022 Maik Vink
+    email                : torcs@free.fr
+    version              : $Id: conformationmenu.h ... $
+
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef _CONFORMATIONMENU_H_
+#define _CONFORMATIONMENU_H_
```

Comment:
Use #pragma once

Reply:
instead of what they do here you mean? (came from speeddreams)